src/styles/components/_subscribe.scss

```diff
@@ -0,0 +1,79 @@
+@import '../settings/colors';
+@import '../tools/breakpoints';
+
+.w-subscribe {
+  display: block;
+  margin-top: 32px;
+  text-align: center;
+  width: 100%;
+}
+
+.w-subscribe__border {
+  border: 2px solid #f1f3f4;
+  border-top-left-radius: 2px;
+  border-top-right-radius: 2px;
+  width: 100%;
+}
+
+.w-subscribe__button {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.w-subscribe__error {
+  color: $WARNING_COLOR;
+}
+
+.w-subscribe__field {
+  display: flex;
+  justify-content: space-between;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 8px;
+  text-align: left;
+}
+
+.w-subscribe__form {
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 720px;
+  width: calc(100% - 32px);
+}
+
+.w-subscribe__input {
+  border: 2px solid #f1f3f4;
+  border-top-left-radius: 2px;
+  border-top-right-radius: 2px;
+  font: inherit;
+  font-size: 16px;
```

Comment:
hmmmm... i wonder if we should define these in a generic/_input.scss file? Seems like we'd want all inputs to have consistent typography. We'd also probably want the font-size to follow whatever body copy is doing at different viewport sizes. This is my fault for not specifying a better default here...

Let's at least not set an explicit font-size. You can leave the font: inherit for now.